src/components/DishGroup/DishGroup.tsx

```diff
@@ -0,0 +1,21 @@
+import { FC } from 'react';
+import { DishGroup as DishGroupProps } from '../../types';
+import classes from './DishGroup.module.scss';
+
+const DishGroup: FC <DishGroupProps> = ({title, dishes}) => {
```

Comment:
maybe call it `Card`, because there is a pretty big possibility that it's gonna be used elsewhere